nin/dasBoot/LayerManager.js

```diff
@@ -124,6 +136,22 @@ LayerManager.prototype.updateActiveLayersList = function(frame, forceUpdate) {
   }
 };
 
+LayerManager.prototype.showCameraPathVisualizations = function(shouldShow) {
+  for(var i = 0; i < this.layers.length; i++) {
```

Comment:
I don't think that it's a part of the any build step in nin. I have the linter set up in my IDE, so that it monitors my code and I can fix code style and errors on the fly, while I code. I find that much smoother than running a command after I've coded my stuff, only to find all the things that need fixing according to the linter.